### ISSUE TITLE
add video and documents support

### ISSUE THREAD

**Issue by Nadimkhan120** [MuhammedKpln/react-native-chatty#23]
please add video and document support

**Comment by MuhammedKpln** [comment]
I've started with the documentation, it is still in progress. For now it is documented the most features you'll need. 

[Documentation](https://muhammedkpln.github.io/react-native-chatty/)

**Comment by MuhammedKpln** [comment]
Well i've already added a types for other media types, it just needs some work to get it working but i need to figure it out how i can do it in a best way. Should documents be downloaded?

**Comment by Nadimkhan120** [comment]
yes, react-native-gifted-chat allow us to add media like image, video, pdf etc

**Comment by MuhammedKpln** [comment]
> yes, react-native-gifted-chat allow us to add media like image, video, pdf etc

Gifted chat is providing a base, but at the ui it needs to provide a renderer prop (video as an example) to render video. As of documents i've seen nothing, but i will investigate, how well i can integrate to chatty.
